CHANGELOG.md

```diff
@@ -23,6 +24,17 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Added `static_lambda` CS rule [#1870](https://github.com/ruflin/Elastica/pull/1870)
 * Added `Elastica\Aggregation\DateRange::setTimezone()` [#1847](https://github.com/ruflin/Elastica/pull/1847)
 * Added endpoint options support to `Elastica\Index::create()` [#1859](https://github.com/ruflin/Elastica/pull/1859)
+* Added `Elastica\Aggregation\Avg::setMissing()` [#1876](https://github.com/ruflin/Elastica/pull/1876)
```

Comment:
Seems to be a few duplicates :-)

Reply:
I think it's more readable this way.
as you wish, I've turned this into a oneliner

Reply:
🤦‍♂️  I really thought it is duplicates but I now realised it wasn't. Sorry, I agree your version before was better :-(